canonical/vanilla-framework
# Add dark theme support to notifications

## Done
- add dark theme support to notifications with examples
- add dark theme to body elements
- update hightlight bars and icon colors with `light`, `paper`, and `dark` theme support

## Fixes 
- https://warthogs.atlassian.net/browse/WD-8572

## QA
- open [demo](https://vanilla-framework-4971.demos.haus/docs/examples/patterns/notifications/variants-dark) for dark theme variants
- open [demo](https://vanilla-framework-4971.demos.haus/docs/examples/patterns/notifications/variants) for light theme variants
- change `body` class from `is-dark` to `is-light` or `is-paper`, and check if background of the page and text color change accordingly
- compare demos with the design proposal below:
![image](https://github.com/canonical/vanilla-framework/assets/90341644/d21b744c-82d4-41ef-bc81-e14970a081ec)


## Screenshots
<img width="1420" alt="Screenshot 2024-02-08 at 7 03 38 PM" src="https://github.com/canonical/vanilla-framework/assets/90341644/c4eddae1-b6fd-4460-8ed1-d29d1d794105">
<img width="1425" alt="Screenshot 2024-02-08 at 3 06 53 PM" src="https://github.com/canonical/vanilla-framework/assets/90341644/363a57b3-34ce-4992-9eeb-3275d550172b">

### If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical/vanilla-framework/wiki/Release-process#pre-release-tasks)
